lsp/server/src/diagnostic/js/adapters-local-change-not-aware.ts

```diff
@@ -45,22 +47,26 @@ export class AdaptersLocalChangeNotAware implements DiagnosticProducer<Node> {
                         start: textDocument.positionAt(item.start as number),
                         end: textDocument.positionAt(item.end as number)
                     },
-                    message: LOCAL_CHANGE_NOT_AWARE_MESSAGE
+                    message: LOCAL_CHANGE_NOT_AWARE_MESSAGE,
+                    code: LOCAL_CHANGE_NOT_AWARE_EXTERNAL_DOC_URL,
```

Comment:
diagnostic renders code and codeDescription as link <a name = '$code$' href = '$codeDescription.href'>